pkg/api/controller.go

```diff
@@ -16,6 +16,10 @@ import (
 	"github.com/gorilla/mux"
 )
 
+const (
+	DBFilePath = "./data/db/ZotSearch.db"
```

Comment:
Is this a separate directory to store the db? zot already has a cache.db it puts at top dir

Reply:
Yes, I made that to store the data, if it has to be cache.db I can change that.

Reply:
No, call it search.db for now.